packages/block-library/src/paragraph/edit.js

```diff
@@ -143,7 +96,6 @@ function ParagraphBlock( {
 			<InspectorControls>
 				<PanelBody title={ __( 'Text Settings' ) } className="blocks-font-size">
 					<FontSizePicker
-						fallbackFontSize={ fallbackFontSize }
```

Comment:
It doesn't seem to be a related change. Well, to be honest, I don't know why it was passed in the first place. You should know better :)

Reply:
It was here because of historic reasons. We used a slider before and we wanted the slider position to respect the current font size the user was seeing even if the user did not explicitly set a size (was using the theme style). Right now with the current font size UI this value is not used at all and although not related to this PR, including this change here allows us to remove the with fallback styles usage.